source/guide_hedgedoc.rst

```diff
@@ -0,0 +1,395 @@
+Then install the dependencies. This step may take a few minutes.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cd hedgedoc
+  [isabell@stardust hedgedoc]$ bin/setup
+  Copying config files...
+  Installing packages...
+  yarn install v1.22.5
+  [1/5] Validating package.json...
+  [2/5] Resolving packages...
+  [3/5] Fetching packages...
+  info There appears to be trouble with your network connection. Retrying...
+  info fsevents@1.2.13: The platform "linux" is incompatible with this module.
+  info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
+  info fsevents@2.1.3: The platform "linux" is incompatible with this module.
+  info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
+  [4/5] Linking dependencies...
+  [5/5] Building fresh packages...
+  Done in 253.95s.
+  yarn install v1.22.5
+  [1/5] Validating package.json...
+  [2/5] Resolving packages...
+  success Already up-to-date.
+  Done in 1.15s.
+
+
+  Edit the following config file to setup HedgeDoc server and client.
+  Read more info at https://github.com/hedgedoc/hedgedoc#configuration-files
+
+  * config.json           -- HedgeDoc config
+  * .sequelizerc          -- db config
+  [isabell@stardust hedgedoc]$
+
+Database
+--------
+
+Create a Database.
+
+.. code-block:: console
+
+  [isabell@stardust hedgedoc]$ mysql --verbose --execute="CREATE DATABASE ${USER}_hedgedoc"
+  --------------
+  CREATE DATABASE isabell_hedgedoc
+  --------------
+
+  [isabell@stardust hedgedoc]$
+
+Configuration
+=============
+
+First we remove the config file that was generated by the setup because it might be very confusing. It was a copy of ``config.json.example`` if you like to have a look at it.
+
+.. code-block:: console
+
+
+  [isabell@stardust hedgedoc]$ rm --verbose config.json
+  removed ‘config.json’
+  [isabell@stardust hedgedoc]$
+
+.. note::
+
+  You can set here some options in json format you can't yet set with environment variables. For other configuration options, check the `configuration documentation`_.
+
+Create ``~/hedgedoc/config.json`` with the following content:
+
+.. code-block:: json
+
+  {}
+
+Edit the file ``~/hedgedoc/.sequelizerc`` with the following content:
+
+.. code-block:: js
+  :emphasize-lines: 7
+
+  var path = require('path');
+
+  module.exports = {
+      'config':          path.resolve('config.json'),
+      'migrations-path': path.resolve('lib', 'migrations'),
+      'models-path':     path.resolve('lib', 'models'),
+      'url':             process.env.CMD_DB_URL
+  }
+
+Generate setting strings
+------------------------
+You will need those highlighted strings at the next step for the ``~/etc/services.d/hedgedoc.ini``.
+
+Generate session secret
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: console
+  :emphasize-lines: 2
+
+  [isabell@stardust ~]$ echo "CMD_SESSION_SECRET=\"$(pwgen 32 1)\""
+  CMD_SESSION_SECRET="somethingSuperRandom"
```

Comment:
Please keep it simple. Just echoing the token will be fine. Please add a description above explaining what this token is for instead of hinting at it in the code.

```suggestion
  [isabell@stardust ~]$ pwgen 32 1
  somethingSuperRandom
```